### PR TITLE
Test run of CSP assessments

### PR DESCRIPTION
**Pull request by mrjoshida**
Test of a few CSP assessments @bakerfranke generated from his giant JSON dump

### CODE REVIEW

File: dashboard/config/scripts/AP_practice_Qs_rqb_86_CSP_4_1_1.multi [hunk 1]
```diff
@@ -0,0 +1,11 @@
+name 'AP_practice_Qs_rqb_86_CSP_4_1_1'
+wrong '<img src="https://codehs.com/uploads/c7d593125a90ac3c790a78d5b5d098af" height="200">'
+wrong '<img src="https://codehs.com/uploads/f9506ef1e6ea5e845699feaac9686a5d" height="200">'
+wrong '<img src="https://codehs.com/uploads/2ffc72ca9ba682bcd443104de50ac8a5" height="200">'
+right '<img src="https://codehs.com/uploads/6b450add4f3ff7310fa1a3c2d6a1b27a" height="200">'
```
Comment (joshlory) [inline dashboard/config/scripts/AP_practice_Qs_rqb_86_CSP_4_1_1.multi]: Can we re-host the images on images.code.org?  Our network requirements only tell administrators to unblock `*.code.org` and everything should work.

Comment (mrjoshida) [reply dashboard/config/scripts/AP_practice_Qs_rqb_86_CSP_4_1_1.multi]: @bakerfranke how many of these include images? Can we ask CodeHS for a dump of them, or are we going to need to pull urls from the JSON?

File: dashboard/config/scripts/AP_practice_Qs_rqb_86_CSP_4_1_1.multi [hunk 2]
```diff
@@ -0,0 +1,11 @@
+name 'AP_practice_Qs_rqb_86_CSP_4_1_1'
+wrong '<img src="https://curriculum.code.org/media/uploads/csp/c7d593125a90ac3c790a78d5b5d098af" height="200">'
+wrong '<img src="https://curriculum.code.org/media/uploads/csp/f9506ef1e6ea5e845699feaac9686a5d" height="200">'
+wrong '<img src="https://curriculum.code.org/media/uploads/csp/2ffc72ca9ba682bcd443104de50ac8a5" height="200">'
+right '<img src="https://curriculum.code.org/media/uploads/csp/6b450add4f3ff7310fa1a3c2d6a1b27a" height="200">'
+markdown <<MARKDOWN
```
Comment (mrjoshida) [inline dashboard/config/scripts/AP_practice_Qs_rqb_86_CSP_4_1_1.multi:6]: @joshlory moved these to curriculum.code.org to unblock testing this process

Comment (bakerfranke) [reply dashboard/config/scripts/AP_practice_Qs_rqb_86_CSP_4_1_1.multi:6]: Okay.  The entire dataset only contains 24 images.  These few files have several of them.  Thanks @mrjoshida for moving these images to curriculum.code.org.  I'll figure out a way to do the others when we push the rest of the set.  Anything blocking this now?

**Comment by mrjoshida** [comment]
@joshlory any concern about merging this so we can figure out if the process is going to work for the larger batch?

**Pull request merged**